server.js

```diff
@@ -2,36 +2,109 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import guests from './data/guests.json'
+import { Guest } from './models/guest'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+// MONGOOSE SETUP
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/guest-list'
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+// SEEDING FOR ADDING NEW DATA
+if (process.env.RESET_DB) {
+  console.log('Resetting database')
+  const seedDatabase = async () => {
+    await Guest.deleteMany({})
+
+    guests.forEach((guestData) => {
+      new Guest(guestData).save()
+    })
+  }
+  seedDatabase()
+}
+
+// PORT & APP SETUP
+const port = process.env.PORT || 8000 // Default 8000, can be overridden e.g PORT=5000 npm run dev
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+// MIDDLEWARES
 app.use(cors())
 app.use(bodyParser.json())
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: 'Service unavailabale' })
+  }
+})
 
-// Start defining your routes here
+// ROUTES FOR GET
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: GET/guests GET/guests/:id GET/guests?name= GET/guests?attending=true/false')
```

Comment:
Awesome that you include usage in your default page.  I believe it's `guest-list`.  You could solve this discrepancy by making the name of your endpoint a `const`